api/tests/integration/tests/inchi/reactions.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+# adjust this path as needed to import env_indigo
+sys.path.append(
+    os.path.normpath(
+        os.path.join(os.path.abspath(__file__), "..", "..", "..", "common")
+    )
+)
+from env_indigo import *  # noqa
+
+# Initialize Indigo and InChI generator
+indigo = Indigo()
+indigo.setOption("ignore-noncritical-query-features", "true")
+indigo_inchi = IndigoInchi(indigo)
+
+# List of SMILES to test
+input_smiles = [
+    "CCN([C@H]1CO[C@H](C[C@@H]1OC)O[C@@H]2[C@H]([C@@H]([C@H](O[C@H]2O[C@H]3C#C/C=C\\C#C[C@@]\\4(CC(=O)C(=C3/C4=C\\CSSC(C)(C)CC(=O)N/N=C(/C)\\C5=CC=C(C=C5)OCCCC(=O)N)NC(=O)OC)O)C)NO[C@H]6C[C@@H]([C@@H]([C@H](O6)C)SC(=O)C7=C(C(=C(C(=C7OC)OC)O[C@H]8[C@@H]([C@@H]([C@H]([C@@H](O8)C)O)OC)O)I)C)O)O)C(=O)C",
```

Comment:
add tests 
r0 -no enhanced
r1 - OR enhanced
r2 - AND enhanced
catalyst - OR
p1 - CIS 
p2 OR AND

output
reaction smiles